cmd/internal/cli/build.go

```diff
@@ -278,28 +278,32 @@ func preRun(cmd *cobra.Command, args []string) {
 	sylabsToken(cmd, args)
 }
 
-// checkTargetCollision makes sure output target doesn't exist, or is ok to overwrite.
-func checkBuildTarget(path string, update bool) bool {
+// checkBuildTarget makes sure output target doesn't exist, or is ok to overwrite.
+// And checks that update flag will update an existing directory.
+func checkBuildTarget(path string) error {
+	if !sandbox && update {
+		return fmt.Errorf("only sandbox update is supported: --sandbox flag is missing")
+	}
 	if f, err := os.Stat(path); err == nil {
 		if update && !f.IsDir() {
-			sylog.Fatalf("Only sandbox updating is supported.")
+			return fmt.Errorf("only sandbox update is supported: %s is not a directory", path)
 		}
 		if !update && !force {
 			reader := bufio.NewReader(os.Stdin)
 			fmt.Print("Build target already exists. Do you want to overwrite? [N/y] ")
 			input, err := reader.ReadString('\n')
 			if err != nil {
-				sylog.Fatalf("Error parsing input: %s", err)
+				return fmt.Errorf("error parsing input: %s", err)
 			}
-			if val := strings.Compare(strings.ToLower(input), "y\n"); val == 0 {
-				force = true
-			} else {
-				sylog.Errorf("Stopping build.")
-				return false
+			if strings.Compare(strings.ToLower(input), "y\n") != 0 {
```

Comment:
I think we have a interactive package for this (`internal/pkg/util/interactive`). Should we use that?

Reply:
Updated